recipes/mayavi/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set version = "4.5.0" %}
+
+package:
+  name: mayavi
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/enthought/mayavi.git
+  git_rev: {{ version }}
+
+build:
+  entry_points:
+    - mayavi2 = mayavi.scripts.mayavi2:main
+    - tvtk_doc = tvtk.tools.tvtk_doc:main
+  skip: True  # [osx or (linux and py34) or win]
```

Comment:
In that case just remove the skip. It will skip in conda-forge anyways, since we are not building Python 3.4, but users will still be able to build it elsewhere if they want to.